src/edit_mode/vi/command.rs

```diff
@@ -161,6 +166,7 @@ impl Command {
             Self::Undo => vec![ReedlineOption::Edit(EditCommand::Undo)],
             Self::DeleteToEnd => vec![ReedlineOption::Edit(EditCommand::CutToLineEnd)],
             Self::AppendToEnd => vec![ReedlineOption::Edit(EditCommand::MoveToEnd)],
+            Self::PrependToStart => vec![ReedlineOption::Edit(EditCommand::MoveToStart)],
```

Comment:
Those should be `MoveToLineEnd` and `MoveToLineStart`. The others will jump to beginning and end of the whole buffer